src/sentry/static/sentry/app/views/dashboardsV2/widgetQueries.tsx

```diff
@@ -133,6 +134,7 @@ class WidgetQueries extends React.Component<Props, State> {
     const {projects, environments} = selection;
 
     if (widget.displayType === 'table') {
+      const tableResults: TableDataWithTitle[] = [];
```

Comment:
This fixes a bug that was popping up in tests where state updates hadn't flushed and response data would be doubled up. Using a closure value prevents stale state being read when there are multiple queries.

Reply:
Does this ever get updated? Seems to always be an empty array

Reply:
You're right, I'll get that fixed I had a `push()` but wanted to avoid in-place mutation, but now it is broken in a different way.